forum/qa-content/qa-page.js

```diff
@@ -568,8 +563,19 @@ function qa_ajax_error()
 			copyCodeBtn.textContent = 'Kopiuj';
 			copyCodeBtn.classList.add('content-copy-btn');
 
-			if (addCopyBtn && window.hasOwnProperty('SyntaxHighlighter'))
-				copyCodeBtn.addEventListener('click', copyToClipboard);
+			if (!!copyToClipboard && window.hasOwnProperty('SyntaxHighlighter'))
+				copyCodeBtn.addEventListener('click', function( ev ) {
+					var eTarget = ev.target;
+
+					var blockOfCodeBar = eTarget.parentNode.parentNode;
+
+					// get block of code content - practically all lines of code inside
+					var codeContent = Array.from( blockOfCodeBar.querySelector( '.code .container' ).children ).map( function( lineOfCode ) {
+						return lineOfCode.textContent  + '\r\n';
+					} ).join( '' );
+

```

Comment:
- Extract `Array.from( blockOfCodeBar.querySelector( '.code .container' ).children )` to variable with some meaningful name
-
```
.map( function( lineOfCode ) {
    return lineOfCode.textContent  + '\r\n';
} ).join( '' );
```
If you join it afterwards, wouldn't this be better?: (it's just a proposition, both should work fine):
`.reduce( function(content, lineOfCoe) { return content + lineOfCode.textContent  + '\r\n' }, '')`

^ it may need to be adjusted, I didn't test it